### ISSUE TITLE
Add google recaptcha in login page

### ISSUE THREAD

**Issue by harshmangalam** [harshmangalam/event-blend-frontend#3]
Add google recaptcha in login page to  protects the login form

login path :- /login

For more details please look into google recaptch docs
https://developers.google.com/recaptcha/docs/v3

**Comment by mohitahlawat2001** [comment]
i want to work on this issue can you assign me this issue.
also want to ask is there any need of api key for adding this?

**Comment by kartikey369-ind** [comment]
Hello @harshmangalam, I noticed the issue . I'd be happy to work on it if you could kindly assign it to me.

**Comment by harshsennnn** [comment]
Hey @harshmangalam can u assign me with this issue

**Comment by mohitahlawat2001** [comment]
i will make necessay changes

**Comment by mohitahlawat2001** [comment]
@Alienimnida  as written in redme, is there more to add here

1. Install dependencies pnpm i
2. Add env variables
3. pnpm run dev

**Comment by mohitahlawat2001** [comment]
@harshmangalam getting this when trying to sign up

**Comment by harshmangalam** [comment]
Have you setup the server locally ?

**Comment by harshmangalam** [comment]
Just mentioned the repo in READMEN you can setup server locally.

**Comment by mohitahlawat2001** [comment]
not able to continue with issue anybody else can take it, sorry for inconvience

**Comment by harshmangalam** [comment]
Thanks @mohitahlawat2001 for your try.

**Comment by harshmangalam** [comment]
Hii @kartikey369-ind are you still interested in this issues ?

**Comment by kartikey369-ind** [comment]
Iam sorry I may not do as iam assigned in other project

**Comment by harshmangalam** [comment]
@Alienimnida are you interested in this issues ?

**Comment by Alienimnida** [comment]
@harshmangalam I can do it but the issue I showed you today is still persisting TT

**Comment by Alienimnida** [comment]
What things do I need to setup the backend could you please tell me exactly?

**Comment by harshmangalam** [comment]
https://github.com/harshmangalam/event-blend-server

**Comment by Alienimnida** [comment]
Yeah I know I have cloned it but getting error due to that Database URL I guess

**Comment by harshmangalam** [comment]
setup local postgres and use inside .env

**Comment by Alienimnida** [comment]
Hello should I take this up as well?

**Comment by harshmangalam** [comment]
Sure @Alienimnida

**Comment by harshmangalam** [comment]
Please take the latest pull before PR and raise for dev instead of main

**Comment by Alienimnida** [comment]
Oh ok got it will take care of that

**Comment by harshmangalam** [comment]
@Alienimnida let me know if you need further help for recaptcha ?

**Comment by Alienimnida** [comment]
Actually I am quite struggling to implement this just give this week if I can't figure out in this week you can unassign me.

**Comment by harshmangalam** [comment]
No problem take your time. Its Okay if its taking time you will learn something new.

**Comment by Alienimnida** [comment]
Yeah that's true. Please check your discord channel tho I wrote a query there.

**Comment by HP200421** [comment]
@harshmangalam can i work on this issue?

**Comment by harshmangalam** [comment]
@Alienimnida i am assigning this issues to @HP200421

**Comment by Alienimnida** [comment]
Yeah yeah thank you for giving me so much time but I couldn't do it sorry for wasting the time 🏳️‍
